metecho/api/jobs.py

```diff
@@ -875,9 +875,9 @@ def refresh_github_users(project: Project, *, originating_user_id):
         for collaborator in repo.collaborators():
             try:
                 # Retrieve additional information for each user by querying GitHub
-                collaborator.name = get_cached_user(
-                    GitHub(repo), collaborator.login
-                ).name
+                collaborator.name = (
+                    get_cached_user(GitHub(repo), collaborator.login).name or ""
+                )
```

Comment:
This is a fix to another error I was seeing where user's where trying to be created with null name values. The except block below attempts to mitigate one version of this, but I was seeing another.